npm-shrinkwrap.json

```diff
@@ -101,6 +101,23 @@
       "from": "any-promise@>=1.1.0 <2.0.0",
       "resolved": "https://registry.npmjs.org/any-promise/-/any-promise-1.3.0.tgz"
     },
+    "appdmg": {
```

Comment:
I don't know why this changes are here, I just added `inquirer` and removed `resin-cli-form` in `package.json`. @jviotti can you tell me if it should be this way?